docs/annotating_code/supported_annotations.md

```diff
@@ -324,6 +324,27 @@ echo Arithmetic::addCumulative(3); // outputs 3
 echo Arithmetic::addCumulative(3); // outputs 6
 ```
 
+### `@psalm-private-set`
```

Comment:
The word „set“ has a lot of meanings: what do you think about „mutate“, „change“ or similar

Reply:
In the description I use "modify". 
It is technically a "set" though and as it's to get rid of "getters and setters", I think "set" does fit.
"change" would be to generic for my taste.

"set", "modify" and "mutate" would all work for me, but I guess it's most important what most people will understand. @muglug What do you think?

Reply:
Oh mutate is probably the best since that’s used elsewhere. So `@psalm-private-mutate`

Reply:
Ok, I've updated the documentation, issue and pull request naming